lib/workers/pr/changelog/release-notes.ts

```diff
@@ -177,12 +214,12 @@ export async function getReleaseNotesMd(
       );
     }
     const fileRes = await http.getJson<{ content: string }>(
```

Comment:
Wouldn't this use the GithubHttp to fetch contents from Gitlab ?

I think this file need some more decupling / refactoring. Maybe we can do this after merge.
The computation of urls should be done in corresponding platform layer otherwise it would be very difficult to support more platforms later

Reply:
I feel like the whole module needs a rewrite. We are also inefficient at caching too (separate caches per-release instead of per-dependency). Let's defer most improvements until after merge.